### PR TITLE
[WIP] DBZ-607 Add Cassandra connector to debezium-incubator (documentation)

### PR DESCRIPTION
**Pull request by jgao54**
Here's an initial draft of the documentation for Cassandra connector. There are a few TODOs throughput the documentation that needs to be finalized, but wanted to make this PR available for reading in the mean time.

### CODE REVIEW

**Comment by alexott** [comment]
Shouldn't the `docs/connectors/index.html.haml` be updated too to include reference to Cassandra connector?

**Comment by gunnarmorling** [comment]
Hey @jgao54, just checking in on this one; do you think you will be able to wrap this up for a release this Thursday?

@alexott, yes, good point. We'll link it from there, too.

**Comment by jgao54** [comment]
@alexott  Thanks for catching that! Will add.

**Comment by gunnarmorling** [comment]
Superseded by #344, where I've added the new connector to the menu and other places as needed. Thanks a lot, @jgao54!

@alexott, thanks again for checking out this PR. If you got any feedback on the connector itself, please let us know, would be great to learn from your insights and exchange. This connector is part of the Beta4 release, out today, so you can easily try it out. Just keep in mind it's still in flux :) Thanks!

File: docs/connectors/cassandra.asciidoc [hunk 1]
```diff
@@ -0,0 +1,982 @@
...
+==== Cassandra Cassandra connector is Stopped for a Duration
+
+Depending on the write load of a table, when a Cassandra connector is stopped for a long time, it risks into hitting the cdc_total_space_in_mb capacity. Once this upper limit is reached, Cassandra will stop accepting writes for this table; which means it is important to monitor this space while running the Cassandra connector. In the worst case scenario when this happens, the best thing to do is to (1) turn off Cassandra connector (2) disable cdc for the table so it stops generating additional writes (although writes to other cdc-enabled tables on the same node could still affect the commitlog file generation given the commit logs are not filtered) (3) remove the recorded offset from the offset file (4) once the capacity is increased or the directory used space is under control, restart the connector so it will rebootstrap the table.
+
+[[deploying-a-connector]]
+== Deploying A Connector
+
+The Cassandra connector should be deployed each Cassandra node in a Cassandra cluster.  The Cassandra connector Jar file takes in a cdc configuration file. See link:#example-configuration[see example configurations] for reference.
```
Comment (qianji) [inline docs/connectors/cassandra.asciidoc:768]: How/where to deploy in the cassandra cluster

**Pull request closed without merging**